### PR TITLE
2.10.6 release deps and changelog

### PR DESCRIPTION
**Pull request by dmsimard**
Built via https://github.com/ansible-community/antsibull/pull/223
Initial build recorded here: https://demo.recordsansible.org/playbooks/2847.html
Rebuild with ansible-base >= 2.10.5 recorded here: https://demo.recordsansible.org/playbooks/2850.html

### CODE REVIEW

File: 2.10/CHANGELOG-v2.10.rst
Comment (dmsimard) [inline 2.10/CHANGELOG-v2.10.rst:33]: Need to fix

Comment (dmsimard) [reply 2.10/CHANGELOG-v2.10.rst]: Fixed in https://github.com/ansible-community/ansible-build-data/pull/55/commits/6cb2fb8c45616fdc777ecac6ec9926d6cec3947c and https://github.com/ansible-community/ansible-build-data/pull/55/commits/b06fde3873f35a15e19e013927be8bca5e8d7c41

**Review by felixfontein — APPROVED** [review]

**Comment by dmsimard** [comment]
Setting as draft so we don't merge while waiting for @abadger to review

**Comment by abadger** [comment]
Looks good.  We'll still have to remember to tag the commit manually, it looks like?

**Comment by dmsimard** [comment]
@abadger yes, tagging 2.10.6 after merging this would be appropriate :)

**Pull request merged**
